### PR TITLE
feat: support for overriding config location via environment variable GOOGLE_API_CERTIFICATE_CONFIG

### PR DESCRIPTION
**Pull request by lcforges**
… GOOGLE_API_CERTIFICATE_CONFIG

### CODE REVIEW

**Comment by clundin25** [comment]
@lcforges thanks! Please sign the Google CLA

**Review by clundin25 — APPROVED** [review]

File: client/util/util_test.go [hunk 1]
```diff
@@ -27,3 +28,12 @@ func TestLoadSignerBinaryPath(t *testing.T) {
 		t.Errorf("Expected path is %q, got: %q", want, path)
 	}
 }
+
+func TestGetConfigFilePathFromEnv(t *testing.T) {
```
Comment (clundin25) [inline client/util/util_test.go:32]: Can you add a test case that checks that an explicit filepath sent to the Cred function is prioritized over the env variable?

File: client/util/util_test.go [hunk 2]
```diff
@@ -15,6 +15,7 @@ package util
 
 import (
 	"testing"
+	"os"
```
Comment (andyrzhao) [inline client/util/util_test.go:17]: FYI, golang convention requires imports to be in alphabetic order (if grouped together) - you can use the command "gofmt -w util_test.go" to format+update the file in-place with the right formatting.

**Review by clundin25 — APPROVED** [review]

File: client/util/util_test.go (hunk 1 above)
Comment (andyrzhao) [reply client/util/util_test.go:32]: client_test.go would probably be where you'd add the test to test the behavior Carl mentioned.

**Review by andyrzhao — COMMENTED** [review]
LGTM overall, nice work!

File: client/util/util_test.go (hunk 2 above)
Comment (lcforges) [reply client/util/util_test.go:17]: Ah okay, that makes sense. I'll be sure to run that command in the future!

File: client/util/util_test.go (hunk 1 above)
Comment (lcforges) [reply client/util/util_test.go:32]: Okay, I will work on that. Thank you!

**Pull request merged**